src/piton/extractors/omop_converter.py

```diff
@@ -293,70 +303,69 @@ def remove_nones(patient: Patient) -> Patient:
     return Patient(patient.patient_id, new_events)
 
 
-# Delta encode rows
-def delta_encode(patient: Patient) -> Patient:
-    last_value: Dict[Tuple[int, datetime.date], Any] = {}
-
+def _move_to_day_end(patient: Patient) -> Patient:
+    """We assume that everything coded at midnight should actually be moved to the end of the day"""
     new_events = []
```

Comment:
Is there a compelling reason not to just edit the `patient.events` object in-place (rather than creating a `new_events` list from scratch)

Reply:
Patients are immutable.